seattle_example/extra_data.py

```diff
@@ -25,7 +25,8 @@
         Column('actual_upload', Float),
         Column('min_rtt', Integer),
         Column('location_type', String),
+	Column('isp_user',String),
         Column('client_ip', BigInteger),
-        Column('cost_of_service', Integer))
+        Column('cost_of_service', String))
```

Comment:
In issue #94 you mention *convert any decimal field to a float (corrects data type for cost of service)*, however this is defining the `cost_of_service` column as a string. I just want to be sure this is expected.